src/Robo/Commands/Blt/UpdateCommand.php

```diff
@@ -120,7 +119,8 @@ public function update(array $options = ['since' => InputOption::VALUE_REQUIRED]
    */
   public function initGitignore() {
     $gitignore = [];
-    $gitignore_file = Path::join($this->getConfigValue('repo.root'), '.gitignore');
+    $filesystem = new Filesystem();
```

Comment:
I think the Path library was ported to Symfony Filesystem: https://github.com/symfony/filesystem/blob/e75960b1bbfd2b8c9e483e0d74811d555ca3de9f/Path.php#L656

Why not use the ported version of this method? Just make sure to add a Composer dependency on Symfony Filesystem (seems like we should have that already).

Reply:
Good idea, not sure why I missed that. Yes it has been ported over.

Updated - https://github.com/acquia/blt/pull/4670/commits/08937afc46c3db4376ca8121831da07420490927